kinetic_maintainer_update/package.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<!--
+  Copyright 2016 The Cartographer Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<package format="2">
+  <name>cartographer</name>
+  <version>0.2.0</version>
+  <description>
+    Cartographer is a system that provides real-time simultaneous localization
+    and mapping (SLAM) in 2D and 3D across multiple platforms and sensor
+    configurations.
+  </description>
+  <maintainer email="cartographer-owners@googlegroups.com">
+    The Cartographer Owners
+  </maintainer>
```

Comment:
Do you have access to that mailing list. If not, please add yourself as a maintainer to receive the buildfarm emails in case of build failures